### ISSUE TITLE
Error after pressing OK in step 7 terminal 2

### ISSUE THREAD

**Issue by abdelrahmanhmed** [alexa/alexa-avs-sample-app#891]
Hi,

I have been trying to install alexa on my raspberry pi 3 model b but after pressing ok in step 7 terminal 2 i get the massage that says
¨We're sorry!
An error occurred when we tried to process your request. Rest assured, we're already working on the problem and expect to resolve it shortly¨ 
can you plz help??

**Comment by ryter--** [comment]
Have you edited the allowed origins and return urls in your security profile?

**Comment by abdelrahmanhmed** [comment]
no should i and if so how?

**Comment by ryter--** [comment]
Yes, you have to if you are using the companionService, which is what you are doing if you are following the installation guide for the raspberry. Otherwise Amazon won't accept the the request for the access token (at least that is how I understand it).
For instructions on how to do that, see step 3 in https://github.com/alexa/alexa-avs-sample-app/wiki/Raspberry-Pi
